packages/samples/headless-react/src/engine.ts

```diff
@@ -13,3 +14,8 @@ export const recommendationEngine = new HeadlessEngine({
   configuration: HeadlessEngine.getSampleConfiguration(),
   reducers: recommendationAppReducers,
 });
+
+export const productRecommendationEngine = new HeadlessEngine({
+  configuration: HeadlessEngine.getSampleConfiguration(),
+  reducers: productRecommendationsAppReducers,
+});
```

Comment:
We need yet another engine for this.

Reply:
Sounds good, we'll set time to revisit this and see if we can do things better,